packages/ui-kit/tsconfig.cjs.json

```diff
@@ -6,5 +6,5 @@
     "module": "CommonJS"
   },
   "include": ["src/**/*.ts", "src/**/*.tsx"],
-  "exclude": ["src/**/*.stories.*", "src/**/*.test.*", "src/**/__test__/**/*", "src/**/testing/**/*"]
```

Comment:
@sashathor I like the refactoring you did, just curious why did you decide to include the `__test__` folders?
Those folders include the `mockHandlers.ts` files, I think we should not include those files in the bundle as they are importing `msw` and this is what is causing the issue.

What is the reason to keep them?

Reply:
@felipecadavid Actually, I've removed them from the `exclude` list since we don't have `__test__` folders anymore.
Here are the actual values (excluding storybook stories, tests, and tests env from the build) for both build configs:
```
"exclude": ["src/**/*.stories.*", "src/**/*.test.*", "src/**/testing/**/*"]
```

Reply:
Right, I didn't notice I was keeping that locally, then it looks good 👍